process faults {2: ({1, 0, 2}, 587476144)}
network faults {}
{8: ("3509A10D7A4D62E1600E393952F584EE87F4E64C54B83B5E726E0BD91E5AF1D1", {4, 5, 3, 1, 6, 2, 0}), 9: ("BA592F7E9389306D1EA65B9EF10FC910DB1423D8D5D383AFDC493EE4C6144710", {6, 5, 2, 4, 1, 0, 3}), 10: ("F944244AE2977AAEA664DE8DB312FA37EB9DB32DAD2C8647EB6FA60C1617B423", {5, 2, 1, 4, 6, 0, 3}), 5: ("E0F043F9FA6A6D7E12AA394A28995BAE77D304912782C2019F21DC7B80C8183B", {6, 0, 2, 4, 5, 3, 1}), 6: ("057E80C5F2917EA9C4F73155A2799A1196BE1CBBF34D9FF71A364247A800CEBB", {4, 6, 5, 2, 3, 1, 0}), 7: ("13E46FFD36208FEAE408DE31C5346FEA2AD091B0AF0EBD74045E18EAD45E1A18", {4, 6, 3, 2, 0, 1, 5})}
7
reason: all committed
done!
